pilot/pkg/config/kube/gateway/conversion_test.go

```diff
@@ -46,6 +47,7 @@ func TestConvertResources(t *testing.T) {
 			}
 			golden := splitOutput(readConfig(t, goldenFile))
 			if diff := cmp.Diff(golden, output); diff != "" {
+				log.Println(output)
```

Comment:
Please remove this if it is no longer needed.

Reply:
Please remove